### PR TITLE
RHDEVDOCS-5558: Document GitOps monitoring dashboards in admin console

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **GitOps version for cherry-picking**: `gitops-docs-1.10`,  `gitops-docs-1.11`
• **JIRA issue**: [RHDEVDOCS-5558](https://issues.redhat.com/browse/RHDEVDOCS-5558)
• **Preview page**: [Monitoring with GitOps dashboards](https://66452--docspreview.netlify.app/openshift-gitops/latest/observability/monitoring/monitoring-with-gitops-dashboards)
• **SME Review**: Completed by @reginapizza 
• **QE review**: Completed by @varshab1210 
• **Peer-review** completed

### CODE REVIEW

**Comment by ekristova** [comment]
Hello @reginapizza @varshab1210 PTAL at this PR and leave feedback. Thank you!

File: observability/monitoring/monitoring-with-gitops-dashboards.adoc
Comment (ekristova) [inline observability/monitoring/monitoring-with-gitops-dashboards.adoc:21]: The link to the installation will be added with this Jira: https://issues.redhat.com/browse/RHDEVDOCS-5560

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by reginaelyse — APPROVED** [review]
LGTM, thank you :)

File: observability/monitoring/monitoring-with-gitops-dashboards.adoc
Comment (adellape) [inline observability/monitoring/monitoring-with-gitops-dashboards.adoc:14]:
```suggestion
* *{gitops-shortname} Components*: View detailed information, such as CPU or memory, for an application-controller, repo-server, server, and other {gitops-shortname} components.
```

Comment (adellape) [inline observability/monitoring/monitoring-with-gitops-dashboards.adoc:14]:
```suggestion
* *{gitops-shortname} Components*: View detailed information, such as cpu or memory, for application-controller, repo-server, server, and other {gitops-shortname} components.
```
Not sure if this is the right solution, but something about the list was reading a little funny to me.

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (adellape) [inline modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]:
```suggestion
. Optional: Choose specific a namespace, cluster, and interval from the *Namespace*, *Cluster*, and *Interval* drop-down lists.
```

**Review by adellape — COMMENTED** [review]
Looks great!

**Comment by abhatt-rh** [comment]
Hi @ekristova The PR still has `do-not-merge/work-in-progress ` label, so I'm not certain if this is ready for a merge. Also, could you specify the OpenShift version for merge? 
Removing the merge related labels for now. Please re-queue when you are ready. Thanks!

**Comment by ekristova** [comment]
Hello @abhatt-rh I apologize I forgot to remove the `merge-review-needed `label after I realized a colleague of mine wanted to look at the PR as well before merging. Thank you for noticing the do-not-merge label and sorry for the inconvenience! 

Also, to clarify the situation with OCP versions, this is not merged into the main, but into `gitops-docs`, and we are cherry-picking it into the `gitops-docs-1.10` and `gitops-docs-1.11` branches as it is mentioned in the first comment. That is because GitOps docs are standalone now. I hope that explains it. Thank you!

**Comment by abhatt-rh** [comment]
> Hello @abhatt-rh I apologize I forgot to remove the `merge-review-needed `label after I realized a colleague of mine wanted to look at the PR as well before merging. Thank you for noticing the do-not-merge label and sorry for the inconvenience!
> 
> Also, to clarify the situation with OCP versions, this is not merged into the main, but into `gitops-docs`, and we are cherry-picking it into the `gitops-docs-1.10` and `gitops-docs-1.11` branches as it is mentioned in the first comment. That is because GitOps docs are standalone now. I hope that explains it. Thank you!

Thank you for the clarification, @ekristova. That helps!

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (Srivaralakshmi) [inline modules/gitops-accessing-gitops-monitoring-dashboards.adoc:20]: @ekristova Add this before the last bullet item: OpenShift Container Platform monitoring is installed on the cluster.

Comment (Srivaralakshmi) [inline modules/gitops-accessing-gitops-monitoring-dashboards.adoc:20]: @ekristova Did you test this step and option with the latest 1.10 GitOps  Operator in the cluster?

Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:20]: @Srivaralakshmi I suppose you mean the step to enable monitoring, yes I tested it and the whole procedure :)

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @Srivaralakshmi Yes I tested this step, but in my cluster version I did not see all these options, I explain it here in this comment: https://docs.google.com/document/d/1QAHaub04AYH3-LiXqAveXmeM2V-ZyFD2Q6t6ECBWr-A/edit?disco=AAAA6eULc4Q
However, the SME in the same comment confirmed that the options are as described in this step, there are added screenshots in the draft where the comment is.

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:20]: @Srivaralakshmi In our documentation, it is mentioned that the monitoring is preinstalled, this is why I did not include it as a prerequisite: https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html#about-openshift-monitoring (it is here in the first sentence)
So do you think it should be a part of the prerequisite anyway?

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (adellape) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc]: Ha, I just realized how silly this recommendation-typo was. 😬 Hopefully the intent of putting an "a" in the correct spot came through either way. 😅

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:20]: Here is my screenshot:
image

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: Hello @reginapizza I had a discussion with Sri about the issue of not seeing the options to choose namespace and cluster while testing the GitOps dashboards. Here is my screenshot: 
image
As you can see, I can only choose an interval. But I saw that you had those options in the sceenshot you provided.  I have tested it with ocp 4.15 image and with rosa as well and I could not see the options in either of those. We need to know why is that a case before we can merge the issue to docs. Can you help me find out what the problem might be? Thank you!

Comment (reginapizza) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @ekristova ok I think I know the issue... when you have multiple argocd instances it seems that it will pick up on that and then give you the namespace dropdown to either choose "all" or the "openshift-gitops" namespace. Try adding another argocd instance in another namespace and see if you can view that dropdown again.

Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @reginapizza I have tried to add multiple argocd instances in other namespaces as you suggested and it still does not work :(  can there be another issue? 
cc: @varshab1210 could you possibly also help me figure out what might be wrong? Thank you!

Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: Hello @Srivaralakshmi I got on a call with @reginapizza to solve the issue, basically what we were missing was to create new argocd instance in a different namespace and deploy an application in that instance. Then the dashboards pick up the data and show the dropdowns for namespaces and clusters.
Here is the screenshot when I added applications to "example" and "openshift-gitops" namespaces:
image.
So, I think the easy solution for our docs would be to add this to the prerequisites:
* You have installed an Argo CD application in your defined namespace, for example, `openshift-gitops`.
WDYT?

Comment (Srivaralakshmi) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @ekristova yes! Add something along these lines: You have created a new Argo CD instance in your defined namespace, for example, `openshift-gitops`, and deployed an application in that instance.

Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @Srivaralakshmi  I think it is also enough to deploy an application on the instance that is created automatically with installation. Then the prereq would be only the creation of an application. I test that theory and make the prerequisite accordingly.

Comment (Srivaralakshmi) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:28]: @ekristova Yes, I remember stumbling on that observation last time when I tested and shared it with you. I just tested again to confirm it. This prerequisite is good enough: You have installed an Argo CD application in your defined namespace, for example, openshift-gitops. Thanks!

**Comment by Srivaralakshmi** [comment]
@ekristova LGTM from my side.

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (ekristova) [inline modules/gitops-accessing-gitops-monitoring-dashboards.adoc:21]: Hello @reginapizza @varshab1210  I added this one line in the prerequisites about deploying an application so I need your approvals again. Everything else is the same. Can you please take a look? Thank you!

Comment (ekristova) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:21]: Hello @adellape can you also please take a look at this one added line so that the peer review is complete again? Thank you!

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by reginaelyse — APPROVED** [review]

File: modules/gitops-accessing-gitops-monitoring-dashboards.adoc
Comment (adellape) [reply modules/gitops-accessing-gitops-monitoring-dashboards.adoc:21]: LGTM!

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**
